dnf-behave-tests/common/output.py

```diff
@@ -36,6 +36,20 @@ def strip_reposync_dnf5(found_lines, line_number):
     if line_number < len(found_lines) and found_lines[line_number].strip() == "Repositories loaded.":
         found_lines.pop(line_number)
 
+def strip_help_dnf4(found_lines, line_number):
+    pass
+
+def strip_help_dnf5(found_lines, line_number):
+    end_of_help_dnf5 = "--disableplugin=PLUGIN_NAME,...        Alias for '--disable-plugin'"
+    if line_number < len(found_lines) and (found_lines[line_number].strip() == "Missing command" or found_lines[line_number].strip() == "Usage:" ):
+        found_lines.pop(line_number)
+
+    while line_number < len(found_lines) and found_lines[line_number].strip() != end_of_help_dnf5:
+        found_lines.pop(line_number)
+
+    if line_number < len(found_lines) and found_lines[line_number].strip() == end_of_help_dnf5:
+        found_lines.pop(line_number)
+
 
 def handle_reposync(expected, found, dnf5_mode):
```

Comment:
If you want to also handle the `<HELP>` here, some renaming needs to be done to keep the naming of methods accurate.